### DISCUSSION TITLE
Auties00 apk-parser dependency not found on cobalt 0.0.4

### DISCUSSION THREAD

**Discussion by juanallendexmarts** [Auties00/WhatsappWeb4j#470] [General]
I'm getting this error when trying to compile my project with gradle. 0.0.3 works just fine, it only pops up with 0.0.4

This new version apparently also requires "aspose-words:22.11", which I added together with the rest:

```
repositories {
    mavenLocal()
    maven {
        url = uri("https://repo1.maven.org/maven2/")
    }

    maven {
        url = uri("https://releases.aspose.com/java/repo/")
    }

    maven {
        url = uri("https://repo.maven.apache.org/maven2/")
    }
}

dependencies {
    ...
    // https://mvnrepository.com/artifact/com.aspose.words/aspose-words
    implementation("com.aspose:aspose-words:22.11")

    implementation("com.github.auties00:cobalt:0.0.4")
    annotationProcessor("com.github.auties00:cobalt:0.0.4")
}
```

    

![image](https://github.com/Auties00/Cobalt/assets/124298626/77744e5d-5f58-43e8-a5b9-093317e77080)

**Comment by lunalna** [comment]
Hi. ApkParser is in @Auties00 Github. To use the repo, just add `maven { url 'https://www.jitpack.io' }` to your repositories.

**Comment by juanallendexmarts** [comment]
With that change I'm getting this error

![image](https://github.com/Auties00/Cobalt/assets/124298626/08f3b404-8482-4384-8977-e42373192195)
